### PR TITLE
👌 IMPROVE: Workflow summary

### LINKED ISSUE #57: 👌 IMPROVE: Workflow Summary
The workflow summary tab is now a little bare bone, and in monospace font for some reason. Just off the top of my head, we can add:

* Title
* Little explanation on what this tab contains.
* Settings from Step2 outlined.
* References to cite
* Dropdown with "methods text"

And made the layout pretty. 😍

### PR DESCRIPTION
**Pull request by mbercx**
Fixes #57 

Change the workflow summary to have two tables with the "WorkChain
setings" and the "Advanced Settings". Fix several pieces in the code to
make sure the output it obtained correctly.

Other changes: 

* Move to using jinja files and CSS to write down these somewhat
  larger HTML parts that can do with some styling.
* Add `isort` to the `pre-commit` config and have it play nice with `black`.
* Switch to using builtin `importlib.resources` instead of the backport
  `importlib_resources`.

### CODE REVIEW

File: aiidalab_qe/node_view.py [hunk 1]
```diff
@@ -6,16 +6,19 @@
...
 
 import json
+
 import ipywidgets as ipw
 import nglview
 from aiida.orm import Node
 from aiidalab_widgets_base import ProcessMonitor, register_viewer_widget
 from aiidalab_widgets_base.viewers import BandsDataViewer, StructureDataViewer
 from ase import Atoms
+from importlib_resources import files
```
Comment (csadorf) [inline aiidalab_qe/node_view.py:16]: Please move this into the standard library block below above line 8. In general I follow convention:
```
<STDLIB IMPORTS>

<3rd PARTY IMPORTS>

<LOCAL IMPORTS>
```
Where each block is sorted alphabetical with "import ..." and then "from ...".

**Review by csadorf — APPROVED** [review]
I have one soft change request, otherwise this LGTM. I didn't review this line by line.

File: aiidalab_qe/node_view.py (hunk 1 above)
Comment (mbercx) [reply aiidalab_qe/node_view.py:16]: 👌 Do you think it makes sense to use tools like `isort`?

Comment (mbercx) [reply aiidalab_qe/node_view.py:16]: https://github.com/PyCQA/isort

Comment (mbercx) [reply aiidalab_qe/node_view.py:16]: Hmm, seems `black` and `isort` don't get along. 😭

**Comment by mbercx** [comment]
@csadorf maybe have a peek before I merge ;)

File: qe.ipynb [hunk 2]
```diff
@@ -16,18 +16,71 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 2,
```
Comment (csadorf) [inline qe.ipynb:19]: Please make sure to submit a notebook with all cells cleared. You need to clear the outputs and then save the notebook prior to the commit.

**Review by csadorf — CHANGES_REQUESTED** [review]

File: qe.ipynb (hunk 2 above)
Comment (mbercx) [reply qe.ipynb:19]: Ah, I see. Yeah, that's one of the tricky things of version controlling `ipynb` files.

**Review by csadorf — APPROVED** [review]
Great, thx! 👍

**Pull request merged**